image/assets/lang/Deutsch.json

```diff
@@ -80,6 +80,10 @@
   "common_minutes_lower": "minuten",
   "common_minutes_lower_genitive": "minuten",
   "common_minute_short": "min",
+  "common_second_lower": "TODO",
+  "common_seconds_lower": "TODO",
+  "common_seconds_lower_genitive": "TODO",
+  "common_second_short": "TODO",
```

Comment:
just do not add translations for languages that are not used. Default English will be taken in their place. If there is none for English - ten i18n keys will be taken i.e. `common_second_lower` 
This is how it works by design :)

Reply:
Fixed